pom.xml

```diff
@@ -29,7 +29,7 @@
   <modules>
     <module>hydrator-common</module>
     <module>core-plugins</module>
-    <module>cassandra-plugins</module>
+<!--    <module>cassandra-plugins</module> TO EVALUATE https://cdap.atlassian.net/browse/CDAP-21051 -->
```

Comment:
Please mark it as TODO. It's easier to track as TODO is picked up by many tools, including IDE